### PR TITLE
update eslint

### LINKED ISSUE #33: Update supported eslint to include v9
I'm working on upgrading TypeScript's eslint version to v9 now that ts-eslint supports it, but this package declares too low of a peer dep on eslint so npm doesn't want to install it without `--legacy-peer-deps`.

Can this package be updated to also say it supports v9? seems to work in my testing.

### PR DESCRIPTION
**Pull request by a-tarasyuk**
Fixes #33

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,32 +25,30 @@
...
+    "lint": "eslint -c eslint.config.mjs",
     "build": "tsc"
   },
   "dependencies": {
     "chalk": "^4.1.2",
     "plur": "^4.0.0"
   },
   "peerDependencies": {
-    "eslint": "^8.3.0"
+    "eslint": "^9.0.0"
```
Comment (jakebailey) [inline package.json:36]: Should this be both versions? Otherwise this is technically a breaking change (the code itself didn't change)

Comment (a-tarasyuk) [reply package.json:36]: ^8.3.0 || ^9.0.0"?

Comment (jakebailey) [reply package.json:36]: Yeah, that's what I'd do.

Comment (a-tarasyuk) [reply package.json:36]: ok, I've changed peerDependencies and published the new version

**Pull request merged**